packages/next/src/build/create-compiler-aliases.ts

```diff
@@ -269,9 +269,8 @@ export function createRSCAliases(
     'react-dom/static$': `next/dist/compiled/react-dom${bundledReactChannel}/static`,
     'react-dom/static.edge$': `next/dist/compiled/react-dom${bundledReactChannel}/static.edge`,
     'react-dom/static.browser$': `next/dist/compiled/react-dom${bundledReactChannel}/static.browser`,
-    // optimizations to ignore the legacy build of react-dom/server in `server.browser` build
+    // optimizations to ignore the legacy build of react-dom/server in `server.edge` build
     'react-dom/server.edge$': `next/dist/build/webpack/alias/react-dom-server-edge${bundledReactChannel}.js`,
-    'react-dom/server.browser$': `next/dist/build/webpack/alias/react-dom-server-browser${bundledReactChannel}.js`,
```

Comment:
This wasn't even doing anything. The legacy APIs were still bundled in our `server.browser` build